src/redux/books/books.js

```diff
@@ -32,29 +32,29 @@ export const fetchBooksSuccess = (payload) => ({
   payload,
 });
 export const fetchBooks = () => async (dispatch) => {
-  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HmsXokvT4r0o4d4gM6bR/books');
+  const response = await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SERz0T6MOlfncfi0umcc/books');
   const bookData = await response.json();
   const books = Object.entries(bookData).map(([key, value]) => ({
     item_id: key,
     title: {
-      title: value[0].title.title,
-      author: value[0].title.author,
+      title: value[0].title,
+      author: value[0].author,
     },
     category: value[0].category,
   }));
   dispatch(fetchBooksSuccess(books));
 };
 export const removeBook = (id) => async (dispatch) => {
   dispatch(remove(id));
-  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/HmsXokvT4r0o4d4gM6bR/books/${id}`, {
+  await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/SERz0T6MOlfncfi0umcc/books/${id}`, {
```

Comment:
- [ ] **Optional:** You have used the API string a couple of times in this file. It could be nice to store it in a variable and use the variable instead. This is cleaner and implement the [DRY principle](https://www.digitalocean.com/community/tutorials/what-is-dry-development#:~:text=DRY%2C%20which%20stands%20for%20'don,of%20abstractions%20and%20avoiding%20redundancy.).